assets/pages/new-contributions-landing/components/SetPassword/SetPassword.jsx

```diff
@@ -2,153 +2,59 @@
-import { type ThankYouPageStage } from '../../contributionsLandingReducer';
-import { setThankYouPageStage, setPasswordHasBeenSubmitted, updatePassword, type Action } from '../../contributionsLandingActions';
-import { ButtonWithRightArrow } from '../ButtonWithRightArrow';
+import SetPasswordForm from './SetPasswordForm';
 
 // ----- Types ----- //
 
 /* eslint-disable react/no-unused-prop-types */
 type PropTypes = {
-  contributionType: Contrib,
-  email: string,
-  password: string,
-  guestAccountCreationToken: string,
-  setThankYouPageStage: (ThankYouPageStage) => void,
-  setPasswordHasBeenSubmitted: () => void,
-  passwordHasBeenSubmitted: boolean,
-  updatePassword: (Event) => void,
-  csrf: CsrfState,
+  paymentMethod: PaymentMethod,
 };
 /* eslint-enable react/no-unused-prop-types */
 
 
 // ----- State Maps ----- //
 
 const mapStateToProps = state => ({
-  contributionType: state.page.form.contributionType,
-  email: state.page.form.formData.email,
-  password: state.page.form.setPasswordData.password,
-  passwordHasBeenSubmitted: state.page.form.setPasswordData.passwordHasBeenSubmitted,
-  guestAccountCreationToken: state.page.form.guestAccountCreationToken,
-  csrf: state.page.csrf,
+  paymentMethod: state.page.form.paymentMethod,
 });
 
-function mapDispatchToProps(dispatch: Dispatch<Action>) {
-  return {
-    setThankYouPageStage: (thankYouPageStage: ThankYouPageStage) => {
-      dispatch(setThankYouPageStage(thankYouPageStage));
-    },
-    setPasswordHasBeenSubmitted: () => {
-      dispatch(setPasswordHasBeenSubmitted());
-    },
-    updatePassword: (event: Event) => {
-      if (event.target instanceof HTMLInputElement) {
-        dispatch(updatePassword(event.target.value));
-      }
-    },
-  };
-}
-
-
-// ----- Functions ----- //
-
-function onSubmit(props: PropTypes): Event => void {
-  return (event) => {
-    props.setPasswordHasBeenSubmitted();
-    event.preventDefault();
-
-    if (!(event.target: any).checkValidity()) {
-      return;
-    }
-
-    // TODO: send user to thank you page with error if password was not set
-    setPasswordGuest(props.password, props.guestAccountCreationToken, props.csrf)
-      .then((response) => {
-        if (response === true) {
-          props.setThankYouPageStage('thankYouPasswordSet');
-        } else {
-          props.setThankYouPageStage('thankYouPasswordNotSet');
-        }
-      });
-  };
-}
-
-
 // ----- Render ----- //
 
 function SetPassword(props: PropTypes) {
+  if (props.paymentMethod === 'DirectDebit') {
+    return (
+      <div className="set-password__content">
+        <h1 className="header">Thank you for a valuable contribution. Your Direct Debit has been set up.</h1>
+        <section className="confirmation">
+          <p className="confirmation__message">{'Look out for an email confirming your recurring payment. This will appear as \'Guardian Media Group\' on your bank statements.'}</p>
```

Comment:
```suggestion
          <p className="confirmation__message">{'Look out for an email within three business days confirming your recurring payment . This will appear as \'Guardian Media Group\' on your bank statements.'}</p>
```